File: KrisHemenway.AndroidApp/app/src/main/java/com/krishemenway/notifications/Notification.kt
package com.krishemenway.notifications

import com.google.gson.annotations.SerializedName
import java.util.*

data class Notification(
    @SerializedName("NotificationId") val notificationId: String,
    @SerializedName("Title") val title: String,
    @SerializedName("Content") val content: String,
    @SerializedName("TypeName") val typeName: String,
    @SerializedName("SentTime") val sentTime: Date
)

File: KrisHemenway.Notifications/Notification.cs
using StronglyTyped.GuidIds;
using System;

namespace KrisHemenway.Notifications
{
	public class Notification
	{
		public Id<Notification> NotificationId { get; set; }

		public string Title { get; set; }
		public string Content { get; set; }

		public string TypeName { get; set; }
		public DateTime SentTime { get; set; }
	}
}
